+nosnoc/+dcs/Gcs.m

```diff
@@ -0,0 +1,67 @@
+classdef Gcs < nosnoc.dcs.Base
+    properties
+        lambda
+
+        f_x
+
+        dims
+
+        f_fun
+        nabla_c_fun
+        c_fun
+        g_comp_path_fun
```

Comment:
why not in base? is this not the user path complementarity constrains? 
If not, what is it? Use more instructive name if not the above?

Reply:
No reason. This should not be there.